src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -96,6 +100,32 @@ const styles = StyleSheet.create({
   },
   ml8: {
     marginLeft: 8
+  },
+  attachmentContainer: {
+    width: '100%',
+    height: 268,
+    borderRadius: 8,
+    overflow: 'hidden',
+    flexDirection: 'row',
+    flexWrap: 'wrap'
+  },
+  image: {
+    width: '100%',
+    height: '100%',
+    resizeMode: 'cover'
+  },
+  moreOverlay: {
+    position: 'absolute',
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    width: '100%',
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  moreText: {
+    fontSize: dimen.normalizeDimen(16),
+    fontFamily: fonts.inter[400],
+    color: colors.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for handling attachments in the chat. These include styles for the attachment container, image, overlay, and text. Ensure these styles are working as expected across different screen resolutions and devices.



<!-- This is an auto-generated comment by OSS CodeRabbit -->